tests/test_filling_fields.py

```diff
@@ -36,405 +34,385 @@
-        self.assertEqual(len(self.person.nickname),
-                         person_nickname_field.max_length)
+        assert isinstance(person.nickname, str)
+        assert len(person.nickname) == person_nickname_field.max_length
 
-    def test_fill_TextField_with_a_random_str(self):
+    def test_fill_TextField_with_a_random_str(self, person):
         person_bio_field = models.Person._meta.get_field('bio')
-        self.assertIsInstance(person_bio_field, fields.TextField)
-
-        self.assertIsInstance(self.person.bio, str)
-
-
-@pytest.mark.skipif(connection.vendor != 'postgresql', reason='PostgreSQL specific tests')
-class TestCIStringFieldsFilling:
-    def test_fill_cicharfield_with_a_random_str(self, person):
-        ci_char_field = models.Person._meta.get_field('ci_char')
-        assert isinstance(ci_char_field, CICharField)
-        assert isinstance(person.ci_char, str)
-        assert len(person.ci_char) == ci_char_field.max_length
+        assert isinstance(person_bio_field, fields.TextField)
 
-    def test_filling_ciemailfield(self, person):
-        ci_email_field = models.Person._meta.get_field('ci_email')
-        assert isinstance(ci_email_field, CIEmailField)
-        assert isinstance(person.ci_email, str)
+        assert isinstance(person.bio, str)
 
-    def test_filling_citextfield(self, person):
-        ci_text_field = models.Person._meta.get_field('ci_text')
-        assert isinstance(ci_text_field, CITextField)
-        assert isinstance(person.ci_text, str)
 
+class TestBinaryFieldsFilling():
 
-class BinaryFieldsFilling(FieldFillingTestCase):
-    def test_fill_BinaryField_with_random_binary(self):
+    def test_fill_BinaryField_with_random_binary(self, person):
         name_hash_field = models.Person._meta.get_field('name_hash')
-        self.assertIsInstance(name_hash_field, fields.BinaryField)
-        name_hash = self.person.name_hash
-        self.assertIsInstance(name_hash, bytes)
-        self.assertEqual(len(name_hash), name_hash_field.max_length)
+        assert isinstance(name_hash_field, fields.BinaryField)
+        name_hash = person.name_hash
+        assert isinstance(name_hash, bytes)
+        assert len(name_hash) == name_hash_field.max_length
 
 
-class DurationFieldsFilling(FieldFillingTestCase):
-    def test_fill_DurationField_with_random_interval_in_miliseconds(self):
+class TestsDurationFieldsFilling():
+
+    def test_fill_DurationField_with_random_interval_in_miliseconds(self, person):
         duration_of_sleep_field = models.Person._meta.get_field('duration_of_sleep')
-        self.assertIsInstance(duration_of_sleep_field, fields.DurationField)
-        duration_of_sleep = self.person.duration_of_sleep
-        self.assertIsInstance(duration_of_sleep, timedelta)
+        assert isinstance(duration_of_sleep_field, fields.DurationField)
+        duration_of_sleep = person.duration_of_sleep
+        assert isinstance(duration_of_sleep, timedelta)
```

Comment:
```suggestion
        assert isinstance(person.duration_of_sleep, timedelta)
```

This comment might be a style comment so totally optional.